cmd/validate.go

```diff
@@ -47,6 +47,8 @@ method.`,
 }
 
 func validateMain(cmd *cobra.Command, args []string) exitCode {
+	plugin.InitInteractive(false)
+
 	// Validate that 'plugin' is a valid plugin and load it
 	plugins, serverOpts, err := serverOptsFor(cmd)
```

Comment:
I'm not sure why `validate` needs all of `wash server`'s arguments.

Reply:
It was an easy way to get access to things we use later. Probably doesn't need everything, but the extra stuff probably doesn't hurt. I'd just look at what's actually used from `plugins` and `serverOpts`. We want `plugins` for the list of available plugins and access to their Root interfaces. We want `serverOpts` for plugin config and whatever `SetupLogging` does.

Reply:
Ah ok. I'll add a comment explaining what you just said.